zsh-activate-py-environment.py

```diff
@@ -35,6 +41,7 @@
     for environment_file in environment_files_list
 }
 
+YAML_ENV_NAME_REGEX = re.compile(r'^\s*name:\s*([-\w]+)')
```

Comment:
This might not cover `.` in environment names, if that's allowed.  Probably should have just done \S*.

Reply:
You'r right! Found something here: https://github.com/conda/conda/blob/e23cd61c12a68149ab9387baea5fb9b4f34b40aa/conda/base/context.py#L1767-L1772
Will fixing this.

Reply:
Checkout: https://github.com/se-jaeger/zsh-activate-py-environment/pull/3